product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -17,7 +17,9 @@ This mechanism has previously meant that if a transaction is pending commit and
 
 Additionally, handling the error could increase replication lag, due to a combination of the time taken to detect the deadlock, the time taken for the client to rollback its transaction, the time taken for indirect garbage collection of the already applied changes, and the time taken to redo the work. 
 
-This is where Parallel Apply’s deadlock mitigation, introduced in PGD 5.2, comes into play. For any transaction, Parallel Apply now looks at upcoming writes, and where there is a potential conflict, places the transaction on hold until the conflict has passed.
+This is where Parallel Apply’s deadlock mitigation, introduced in PGD 5.2, comes into play. For any transaction, Parallel Apply looks at transactions already scheduled for any row that the current transaction wants to write. If it finds one, the row is marked as needing to wait until the other transaction is committed before applying its change to the row.
```

Comment:
do you think if we just put "row (tuple)" behind the first reference it would bridge between the 2 sections or do you think we should just change tuple in 72 to be row

Reply:
```suggestion
This is where Parallel Apply’s deadlock mitigation, introduced in PGD 5.2, comes into play. For any transaction, Parallel Apply looks at transactions already scheduled for any row (tuple) that the current transaction wants to write. If it finds one, the row is marked as needing to wait until the other transaction is committed before applying its change to the row.
```